docs/src/ladder/classic-ladder.adoc

```diff
@@ -605,20 +605,21 @@ digit number eg. %Q3, or %B123. Q coils control HAL out pins, e.g. if
 %Q15 is energized then HAL pin classicladder.0.out-15 will be true. B
 coils are internal coils used to control program flow.
 
-* 'N.O. COIL' - (a relay coil.) When coil is energized it's N.O. contact
+* 'N.O. COIL' - (a relay coil.) When coil is energized its N.O. contact
```

Comment:
Isn't the original one the correct one? It is --> it's ?

Reply:
"it's" would be the correct use